sources/packages/web/src/views/institution/locations/active-applications/ViewSubmittedApplicationScholasticStanding.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Report a change"
+        subTitle="View Application"
+        :routeLocation="goBackRouteParams"
+      />
+    </template>
+    <scholastic-standing-form :initialData="initialData" :readOnly="true" />
+  </full-page-container>
+</template>
+<script lang="ts">
+import ScholasticStandingForm from "@/components/common/ScholasticStandingForm.vue";
+import { computed, onMounted, ref } from "vue";
+import { ScholasticStandingService } from "@/services/ScholasticStandingService";
+import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
+import { RouteLocationRaw } from "vue-router";
+import { useFormatters } from "@/composables";
+import { ScholasticStandingSubmittedDetailsAPIOutDTO } from "@/services/http/dto";
+
+export default {
+  components: {
+    ScholasticStandingForm,
+  },
+  props: {
+    scholasticStandingId: {
+      type: Number,
+      required: true,
+    },
+    locationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const initialData = ref({} as ScholasticStandingSubmittedDetailsAPIOutDTO);
+    const { dateOnlyLongString } = useFormatters();
+
+    onMounted(async () => {
+      const applicationDetails =
+        await ScholasticStandingService.shared.getScholasticStanding(
+          props.scholasticStandingId,
+        );
+
+      initialData.value = {
```

Comment:
If the logic to adapt the date formats is executed in other places it can be moved to the service layer and resolved there to have the Vue "pages" simplified. For instance, the same that was done in the below:

sources\packages\web\src\services\StudentService.ts
![image](https://user-images.githubusercontent.com/61259237/174394574-591628e0-1f13-461e-a97c-b810be3c68ff.png)